src/mxarray.jl

```diff
@@ -3,15 +3,15 @@
 type MxArray
     ptr::Ptr{Void}
     own::Bool
-    
+
     function MxArray(p::Ptr{Void}, own::Bool)
         p == C_NULL && error("NULL pointer for MxArray.")
         self = new(p, own)
         if own
             finalizer(self, release)
         end
         return self
-    end    
+    end
```

Comment:
And as I said before, these format changes make it much harder to review. (White space changes can be filtered out with `w=0` so it's marginally better than changing between `if` and `&&` or renaming variables)